app/models/changeset/pull_request.rb

```diff
@@ -127,6 +127,10 @@ def message
     nil
   end
 
+  def lead_time
+    @data['updated_at'] - @data['created_at']
```

Comment:
If I let a PR sit for a week that does not count into lead-times ?
... link to a definition of lead-time too, so explain what this is used for/means